libs/features/mas/web-components/src/global.css.js

```diff
@@ -68,6 +71,9 @@ styles.innerHTML = `
     --merch-color-focus-ring: #1473E6;
     --merch-color-grey-60: var(--spectrum-global-color-gray-600, #6D6D6D);
     --merch-color-grey-80: #2c2c2c;
+    /* beware, 'gray' not 'grey'*/
```

Comment:
I think the grey comment is not needed. but message is understood.

Reply:
yeah i was wondering if it's helpful or not..
i had a bit of pain noticing the diff, so decided to add a note for the next person

Reply:
removed the message and actually renamed it to 'grey' to be consistent.